src/Classes/LinkRenderers/NavLinkRenderer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace LaraZeus\Sky\Classes\LinkRenderers;
+
+use Illuminate\Database\Eloquent\Model;
+
+abstract class NavLinkRenderer
+{
+    public static string $renders;
+
+    public function __construct(
+        protected array $item
+    ) {
+    }
+
+    // TODO: something to control these classes for end-user?
+    public static string $activeClasses = 'border-b border-b-secondary-500 text-secondary-500';
```

Comment:
As mentioned - having something to control this could be nice, tho not necessary immediately.